File: stage_0_mahfuz.py
def hamming(slack,twitter):

    distance = 0

    L = len(slack)
    for i in range(L):

        if slack[i] != twitter[i]:
            distance += 1

    return distance

name="Md. Mahfuzur Rahman Bhuyan"
email="mdmahfuzurrahmanbhuyan@gmail.com"
slackid="@Mahfuz"
biostack="Python"
twitterid="@RobyRahman13"
print('Name :',name)
print('Email :',email)
print('Slack ID:',slackid)
print('Biostack:', biostack)
print('Twitter ID:',twitterid)
print("Hamming Distance between Slack and Twitter:",hamming(slackid,twitterid))

File: stage_0_mahfuz.pl
#!/usr/bin/perl
use strict;
use warnings;

my $name = "Name: Md. Mahfuzur Rahman Bhuyan\n";
my $email = "Email: mdmahfuzurrahmanbhuyan\@gmail.com\n";
my $slack ="Slack: \@Mahfuz\n";
my $biostack ="Biostack: Genomics\n";
my $twitter ="Twitter: \@RobyRahman13\n";
my $string1 ="Mahfuz";
my $string2 ="RobyRahman13";


print $name;
print $email;
print $slack;
print $biostack;
print $twitter;
print "Hamming Distance: ",hd( $string1, $string2 ),"\n";

sub hd{ length( $_[ 0 ] ) - ( ( $_[ 0 ] ^ $_[ 1 ] ) =~ tr[\0][\0] ) }